src/Config/packages.aget

```diff
@@ -2,13 +2,13 @@
   "nuget": {
     "references": {
       "DynamicLanguageRuntime" : "1.2.2",
-      "DynamoVisualProgramming.Core" : "2.6.0.8481",
-      "DynamoVisualProgramming.DynamoCoreNodes" : "2.6.0.8481",
-      "DynamoVisualProgramming.DynamoServices" : "2.6.0.8481",
-      "DynamoVisualProgramming.Tests" : "2.6.0.8481",
-      "DynamoVisualProgramming.WpfUILibrary" : "2.6.0.8481",
-      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.6.0.8481",
-      "DynamoVisualProgramming.DynamoSamples" : "2.6.0.8481",
+      "DynamoVisualProgramming.Core" : "2.6.1.8786",
+      "DynamoVisualProgramming.DynamoCoreNodes" : "2.6.1.8786",
+      "DynamoVisualProgramming.DynamoServices" : "2.6.1.8786",
+      "DynamoVisualProgramming.Tests" : "2.6.1.8786",
+      "DynamoVisualProgramming.WpfUILibrary" : "2.6.1.8786",
+      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.6.1.8786",
+      "DynamoVisualProgramming.DynamoSamples" : "2.6.1.8786",
       "Greg" : "1.2.7177.24952",
       "GregRevitAuth" : "1.0.7057.20655",
```

Comment:
![image](https://user-images.githubusercontent.com/62596335/165442444-7aea7646-3e35-4a05-b92b-64dfe9d7257c.png)
2.6.1.8786's interfaces are same with 2.6.2.15705
so we can use 2.6.1.8786 to build DynamoRevit.